src/workers/parser/getSymbols.js

```diff
@@ -159,10 +160,12 @@ function extractSymbols(source: Source) {
 
       if (t.isCallExpression(path)) {
         const callee = path.node.callee;
+        const args = path.node.arguments;
```

Comment:
We likely just want to store the value

`path.node.arguments.map(arg => arg.value)`

Reply:
If we do this, then we can't confirm that the expression's name is 'require', I'm not sure how many other functions would have a value of 'react', what do you think? Any that do contain a function like this probably are a react file anyway right 😛

Reply:
I'm confused. Wouldn't we have the call expression name and the list of arguments?

Reply:
Yeah that's my bad, changing it now

Reply:
this still needs to be changed, right?

Reply:
I used just the value lower down to try and save a bit of performance because it does a map, is there any benefit to changing it here?

Reply:
@codehag @jasonLaster do you want me to make this change or is it good?

Reply:
it looks good